docker-compose.fake-fhir.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+services:
+  fhir:
+    image: "hapiproject/hapi:v5.3.0"
+    ports:
+      - "8090:8080"
+    environment:
+      spring.datasource.url: "jdbc:postgresql://fhir-db:5432/hapi_r4"
+      spring.datasource.username: admin
+      spring.datasource.password: admin
+      spring.datasource.driverClassName: org.postgresql.Driver
+      hapi.fhir.subscription.resthook_enabled: "true"
+      hapi.fhir.subscription.websocket_enabled: "true"
+      hapi.fhir.client_id_strategy: ANY
+  fhir-db:
+    image: postgres
```

Comment:
Could the DB be replaced with a different one (e.g. MongoDb)? If so, would that influence the approach we take in structuring our data schema? Can we be confident that we will use a relational database in production?

Reply:
In theory the API should be the same regardless of the backend that is hosting it! Though each provider may implement the specification in a different way